### PR TITLE
Bump version and point users to official README

### PR DESCRIPTION
**Pull request by leandrocr**
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kops/2188)

### CODE REVIEW

File: docs/addons.md [hunk 1]
```diff
@@ -42,11 +42,11 @@ kubectl create -f https://raw.githubusercontent.com/kubernetes/kops/master/addon
 Please note that kops installs a Route53 DNS controller automatically (it is required for cluster discovery).
 The functionality of the route53-mapper overlaps with the dns-controller, but some users will prefer to
 use one or the other.
-[README for the included dns-controller](https://github.com/kubernetes/kops/blob/master/dns-controller/README.md)
+[Official README for Addon](https://github.com/wearemolecule/route53-kubernetes/blob/master/README.md)
```
Comment (yissachar) [inline docs/addons.md]: We may want to include the README link for the Route53-mapper, but we shouldn't remove the link to the DNS controller README.

File: docs/addons.md [hunk 2]
```diff
@@ -42,11 +42,11 @@ kubectl create -f https://raw.githubusercontent.com/kubernetes/kops/master/addon
...
 
 route53-mapper automates creation and updating of entries on Route53 with `A` records pointing
 to ELB-backed `LoadBalancer` services created by Kubernetes. Install using:
 
 ```
-kubectl apply -f https://raw.githubusercontent.com/kubernetes/kops/master/addons/route53-mapper/v1.2.0.yml
+kubectl apply -f https://raw.githubusercontent.com/kubernetes/kops/master/addons/route53-mapper/v1.3.0.yml
```
Comment (yissachar) [inline docs/addons.md]: This doesn't seem to exist?

**Comment by leandrocr** [comment]
@k8s-ci-robot I signed it!

File: docs/addons.md (hunk 2 above)
Comment (justinsb) [inline docs/addons.md]: I think I just merged it - sorry for delay

File: docs/addons.md (hunk 1 above)
Comment (justinsb) [inline docs/addons.md]: Are you OK to make this change @leandrocr ?

File: docs/addons.md (hunk 2 above)
Comment (leandrocr) [inline docs/addons.md]: @yissachar would you mind reviewing my changes plz? cheers!

**Comment by yissachar** [comment]
@k8s-bot ok to test

**Pull request merged**